conda.recipe/conda_build_config.yaml

```diff
@@ -25,9 +25,9 @@
 # See https://sifive.atlassian.net/wiki/x/0IBBJw
 # for details
 macos_min_version:             # [osx]
-  - 10.13                      # [osx]
+  - 12.6                       # [osx]
 MACOSX_DEPLOYMENT_TARGET:      # [osx]
-  - 10.13                      # [osx]
+  - 12.6                       # [osx]
```

Comment:
I think this has maybe changed some since we set it up, the upstream conda-forge guys added support for ARM architecture macos machines and `macos_min_version` doesn't seem to be a thing anymore.  We might want to remove `macos_min_version` in this PR.

NOTE: the stuff we're overriding is currently defined https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/1f2e2a865946512616336f81a87c1c05f145127d/recipe/conda_build_config.yaml#L111-L116

Reply:
Thanks!  Yeah, I see it.  Working with you offline to figure this out.